test/e2e/helm/kuma_helm_upgrade.go

```diff
@@ -52,8 +53,20 @@ func UpgradingWithHelmChart() {
 
 			k8sCluster := cluster.(*K8sCluster)
 
-			err = k8sCluster.UpgradeKuma(core.Standalone, WithHelmReleaseName(releaseName))
+			err = k8sCluster.UpgradeKuma(core.Standalone, WithHelmReleaseName(releaseName), WithHelmChartPath(Config.HelmChartPath))
 			Expect(err).ToNot(HaveOccurred())
+
+			// when
+			out, err := k8s.RunKubectlAndGetOutputE(
+				k8sCluster.GetTesting(),
+				k8sCluster.GetKubectlOptions(),
+				"get", "crd", "meshtrafficpermissions.kuma.io", "-oyaml",
+			)
+
+			// then CRD is upgraded
```

Comment:
A better test might be to patch the existing CRD before running `UpgradeKuma` and make sure that the edit was removed?

Reply:
It might, but I find this to be good enough.